pkgs/by-name/gr/grun/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk2
+, pkg-config
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "grun";
+  version = "0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "lrgc";
+    repo = "grun";
+    rev = "release_0_9_3";
```

Comment:
```suggestion
    rev = "release_${lib.replaceStrings [ "." ] [ "_" ] version}";
```

Reply:
Not really if it's the final version. But then `rec` is unused, would you please remove it?